### ISSUE TITLE
Restore (or remove) latest version fetching for deis cli curl | bash

### ISSUE THREAD

**Issue by sgoings** [deis/deis.io#175]
It looks like after the move to store binaries in GCS we aren't using the `get_latest_version` function in the bash script that's called in the `curl | bash` cli installation process. We should either remove this logic or rewrite it to work with GCS (instead of Bintray).

**Comment by bacongobbler** [comment]
Indeed, especially now that we've implemented `-latest` logig when building the clients. :)

**Comment by kmala** [comment]
do we want the script to download the latest binary or the latest release binary because latest binary is the one pushed by merge to master?

**Comment by Joshua-Anderson** [comment]
Latest release binary. The latest version binary is much more likely to have issue or a version mismatch with the controller.
